README.rst

```diff
@@ -13,27 +13,27 @@ This repository is a template for Open edX micro-frontend applications. It is fl
 After Copying The Template
 --------------------------
 
-You'll want to do a find-and-replace to replace all instances of ``frontend-template-application`` with the name of your new repository.  Also edit index.html to replace "Application Template" with a friendly name for this application that users will see in their browser tab.
+You'll want to do a find-and-replace to replace all instances of ``frontend-app-communications`` with the name of your new repository.  Also edit index.html to replace "Application Template" with a friendly name for this application that users will see in their browser tab.
```

Comment:
Should we delete this entire `After copying the template section?` or replace the entire README now? Or do you think we should handle this in a later PR?